avro-builder/src/main/java/com/linkedin/avroutil1/builder/operations/codegen/OperationContext.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.builder.operations.codegen;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Set;
+
+
+/**
+ * common context to all {@link com.linkedin.avroutil1.builder.operations.Operation}s run during an execution
+ */
+public class OperationContext {
+  private Set<File> avroFiles;
```

Comment:
we want the parsed AvroSchemas available as well, likely even more important than the "raw" input files.

Reply:
As I mentioned in the description, adding avroSchemas will work if we use our own codegen but not if we do vanilla `org.apache.avro` codegen. How do you want to handle the case where we do Vanilla avro codegen? 

A strawman approach would be to add both `apacheAvroSchemas` and `avroUtilSchemas` to the context and plugins should expect only one of them to be defined. The issue then is that plugins have to handle both cases.

Reply:
our own model should be considered part of the builder API.
there should also be utility code to convert from our own AvroSchema to vanilla org.apache.avro.Schema instances